### PR TITLE
Instructors

### PR DESCRIPTION
**Pull request by madhavbanjade**
Instructors API Done...

### CODE REVIEW

File: models/instructor.model.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+class Instructor extends Model {
+  //Define associations if necessary here
+
+  static associate(models) {
+    //association can define here
+  }
+}
+
+Instructor.init(
+  {
```
Comment (Mingtindu) [inline models/instructor.model.js:13]: Add id field

File: src/controllers/instructor.controller.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+  const { Name, Email, Phone } = req.body;
+
+  if (
+    ![Name, Email, Phone].every(
+      (field) => typeof field === "string" && field.trim() !== ""
+    )
+  ) {
+    return res.status(400).json({ message: "All fields are required" });
+  }
+
```
Comment (Mingtindu) [inline src/controllers/instructor.controller.js:15]: add one block of code to check if the instructor is already exist or not 
user will be able to create instructor only if it is  not exist in the database

File: src/controllers/instructor.controller.js [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+    res.status(200).json({ message: "All Instructor read successfully" });
+  } catch (error) {
+    res.status(400).json({ message: "All user could not read successfully" });
+  }
+};
+
+// read individual data of instructor
+const getSpecificInstructor = asyncHandler(async (req, res) => {
+  const id = req.params.id; // getting the id .of instructor from params
+  // console.log(id);
```
Comment (Mingtindu) [inline src/controllers/instructor.controller.js:47]: remove comment

File: src/controllers/instructor.controller.js [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+  }
+
+  try {
+    const instructor = await Instructor.create({
+      Name,
+      Email,
+      Phone,
+    });
+    res
+      .status(200)
```
Comment (Mingtindu) [inline src/controllers/instructor.controller.js:23]: change status to 201

**Review by Mingtindu — CHANGES_REQUESTED** [review]

**Review by Mingtindu — APPROVED** [review]

**Comment by madhavbanjade** [comment]
instructors and schedules

**Pull request merged**